src/steps/bundle.ts

```diff
@@ -1,39 +1,23 @@
 import { NexeCompiler } from '../compiler'
 import { readFileAsync, writeFileAsync } from '../util'
 import { resolve, relative } from 'path'
-import NativeModulePlugin from '../bundling/fuse-native-module-plugin'
 import { NexeOptions } from '../options'
+import resolveFiles from 'resolve-dependencies'
 
-function createBundle(options: NexeOptions) {
-  const { FuseBox, JSONPlugin, CSSPlugin, HTMLPlugin, QuantumPlugin } = require('fuse-box')
-  const plugins: any = [JSONPlugin(), CSSPlugin(), HTMLPlugin(), NativeModulePlugin(options.native)]
-  if (options.compress) {
-    plugins.push(
-      QuantumPlugin({
-        target: 'server@esnext',
-        uglify: true,
-        bakeApiIntoBundle: options.name
-      })
-    )
-  }
-  const cwd = options.cwd
-  const fuse = FuseBox.init({
-    cache: false,
-    log: Boolean(process.env.NEXE_BUNDLE_LOG) || false,
-    homeDir: cwd,
-    sourceMaps: false,
-    writeBundles: false,
-    output: '$name.js',
-    target: 'server@esnext', //try not to transpile
-    plugins
-  })
-  const input = relative(cwd, resolve(cwd, options.input)).replace(/\\/g, '/')
-  fuse.bundle(options.name).instructions(`> ${input}`)
-  return fuse.run().then((x: any) => {
-    let output = ''
-    x.bundles.forEach((y: any) => (output = y.context.output.lastPrimaryOutput.content.toString()))
-    return output
+function makeRelative(cwd: string, path: string) {
+  return './' + relative(cwd, path)
+}
+
+let producer = async function(compiler: NexeCompiler) {
+  const { cwd, input } = compiler.options
+  const { files } = await resolveFiles({ entries: [compiler.options.input], cwd, strict: false })
+  Object.keys(files).forEach(x => {
```

Comment:
Not to comment on your style, especially since I don't have much leeway here. But I feel it's good practice to avoid variables like `x` or such, even if it's something as wasteful as `name` or `path`. It's much more readable.